.github/workflows/unit-tests.yml

```diff
@@ -23,21 +23,21 @@ jobs:
       matrix:
         include:
           # MacOSX
-          - { compiler: clang,    os: macos-12,     buildType: Debug, cxxStandard: 14 }
+          - { compiler: clang,    os: macos-12,     buildType: Debug }
           # Linux GCC
           # GCC 9 is known to show a few warnings that GCC 10 has "fixed", make sure this doesn't happen for us
-          - { compiler: gcc-9,    os: ubuntu-20.04, buildType: Debug }
-          - { compiler: gcc-10,   os: ubuntu-20.04, buildType: Debug, cxxStandard: 14, coverage: true }
-          - { compiler: gcc-10,   os: ubuntu-20.04, buildType: Debug, cxxStandard: 17 }
-          - { compiler: gcc-10,   os: ubuntu-20.04, buildType: Release, cxxStandard: 14 }
+          - { compiler: gcc-9,    os: ubuntu-20.04, buildType: Debug, boostVersion: 1.69.0 }
+          - { compiler: gcc-10,   os: ubuntu-20.04, buildType: Debug, boostVersion: 1.69.0, coverage: true }
+          - { compiler: gcc-10,   os: ubuntu-20.04, buildType: Release }
+          - { compiler: gcc-11,   os: ubuntu-22.04, buildType: Debug, boostVersion: 1.74.0 }
           # Linux clang
-          - { compiler: clang-11, os: ubuntu-20.04, buildType: Debug, cxxStandard: 14, externalSanitizer: true }
-          - { compiler: clang-11, os: ubuntu-20.04, buildType: Debug, cxxStandard: 17 }
-          - { compiler: clang-11, os: ubuntu-20.04, buildType: Release, cxxStandard: 14 }
+          - { compiler: clang-10, os: ubuntu-20.04, buildType: Debug, boostVersion: 1.69.0, externalSanitizer: true }
+          - { compiler: clang-10, os: ubuntu-20.04, buildType: Release }
+          - { compiler: clang-14, os: ubuntu-22.04, buildType: Debug, boostVersion: 1.74.0 }
```

Comment:
I'd add a comment here why Boost 1.74 is used

Reply:
I wrote it down now in the comments